source/class/qx/ui/treevirtual/SimpleTreeDataModel.js

```diff
@@ -907,9 +907,12 @@ qx.Class.define("qx.ui.treevirtual.SimpleTreeDataModel", {
      *   {@link SimpleTreeDataModel}.  Each property value will be assigned
      *   to the corresponding property of the node specified by nodeId.
      *
+     * @param suppressRedraw {Boolean}
+     *    If true then prevents redraw upon expand/contract. Useful if expanding/contracting many
```

Comment:
This doc string is a bit restrictive, I think. This change is making this new parameter available for general use, so how about, `If true then prevents redraw; it becomes the caller's responsibility to call `setData()` subsequently, to cause a redraw.` ?

Reply:
Amended param description